file-share-service-desktop-client/Modules/Auth/AuthenticateView.xaml

```diff
@@ -16,33 +25,44 @@
             <ColumnDefinition Width="*" />
         </Grid.ColumnDefinitions>
 
-        <StackPanel
-            VerticalAlignment="Center"
+        <StackPanel HorizontalAlignment="Left" VerticalAlignment="Top">
+            <Image Stretch="UniformToFill" StretchDirection="Both" Source="/Modules/Auth/image/resources/UKHO-1400x240.jpg" />
+        </StackPanel>
+        <StackPanel Margin=" 0 30"
+            VerticalAlignment="Bottom"
             HorizontalAlignment="Center">
             <TextBlock FontSize="28"
                        TextWrapping="Wrap"
-                       MaxWidth="700"
-                       Margin="0 15">
+                       MaxWidth="1000"
+                       Margin="0 5">
                 The ADMIRALTY File Share Service allows you to search and download files.
             </TextBlock>
-            <Image Source="../../Views/resized_0005_Web.jpg"></Image>
+            
             <Label FontSize="36"
                    FontWeight="Medium"
-                   Margin="0 15">Sign in to access your files</Label>
+                   Margin="0 5">Sign in to access your files</Label>
             <Label FontSize="18"
-                   Margin="0 5 0 0">You need to sign in with your account access the File Share Service.</Label>
-            <ComboBox
-                Margin="0 8"
+                   Margin="0 5">You need to sign in with your account access the File Share Service.</Label>
+
+            <Grid>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition></ColumnDefinition>
+                    <ColumnDefinition></ColumnDefinition>
+                </Grid.ColumnDefinitions>
+                <Grid.RowDefinitions>
+                    <RowDefinition></RowDefinition>
+                    <RowDefinition></RowDefinition>
```

Comment:
You've defined two row definitions here, but then there is nothing in the second Row? (nothing with `Grid.Row="1"`). If that is intentional, can we remove the whole `<Grid.Rowdefinitions>` block please.

Reply:
Done the required changes Kindly recheck